src/main/java/org/team1540/bunnybotTank2023/io/turret/TurretIOReal.java

```diff
@@ -0,0 +1,53 @@
+package org.team1540.bunnybotTank2023.io.turret;
+
+import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.controls.PositionVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.ForwardLimitValue;
+import com.ctre.phoenix6.signals.ReverseLimitValue;
+import edu.wpi.first.math.geometry.Rotation2d;
+import org.team1540.bunnybotTank2023.Constants;
+
+public class TurretIOReal implements TurretIO {
+
+    // fields ^-^
+    private final TalonFX motor = new TalonFX(Constants.TurretConstants.MOTOR_ID);
+    final PositionVoltage positionVoltage = new PositionVoltage(0);
+    private Rotation2d setPosition = new Rotation2d();
+
+    // constructor
+    TurretIOReal() {
+        // robot init, set slot 0 gains
+        var slot0Configs = new Slot0Configs();
+        slot0Configs.kP = Constants.TurretConstants.kP;
+        slot0Configs.kI = Constants.TurretConstants.kI;
+        slot0Configs.kD = Constants.TurretConstants.kD;
+
+        motor.getConfigurator().apply(slot0Configs, 0.050);
+
+        positionVoltage.Slot = 0;
```

Comment:
You shouldn't need to specify the slot for the control request, as I believe it defaults to slot 0. If you want to explicitly declare the slot, I would probably do this in line with the field declaration (i.e. `PositionVoltage positionVoltage = new PositionVoltage(0).withSlot(0);`)